### PR TITLE
MAINT: Updates for 0.1 alpha release

### LINKED ISSUE #1134: Brand new environment cannot access studies
If I starta  brand new environment with the following command:

```bash
qiita_env make --load-ontologies --download-reference --add-demo-user
```

And then create a studyk, I cannot view it i.e. I cannot go to http://localhost:21174/study/description/10000 as I get the following traceback:

```python
Traceback (most recent call last):
  File "/Users/yoshikivazquezbaeza/.virtualenvs/alpha-0.1-rc2/lib/python2.7/site-packages/tornado/web.py", line 1141, in _when_complete
    callback()
  File "/Users/yoshikivazquezbaeza/.virtualenvs/alpha-0.1-rc2/lib/python2.7/site-packages/tornado/web.py", line 1162, in _execute_method
    self._when_complete(method(*self.path_args, **self.path_kwargs),
  File "/Users/yoshikivazquezbaeza/.virtualenvs/alpha-0.1-rc2/lib/python2.7/site-packages/tornado/web.py", line 2297, in wrapper
    return method(self, *args, **kwargs)
  File "/Users/yoshikivazquezbaeza/.virtualenvs/alpha-0.1-rc2/lib/python2.7/site-packages/qiita_pet/handlers/analysis_handlers.py", line 227, in get
    info = Analysis(self.current_user.default_analysis).summary_data()
  File "/Users/yoshikivazquezbaeza/.virtualenvs/alpha-0.1-rc2/lib/python2.7/site-packages/qiita_db/user.py", line 349, in default_analysis
    return conn_handler.execute_fetchone(sql, [self._id])[0]
TypeError: 'NoneType' object has no attribute '__getitem__'
```

@squirrelo, can you have a look?

### PR DESCRIPTION
**Pull request by ElDeveloper**
- Fixes a few errors in setup.py
- Moves the README file to be RST so pypi can render it.
- Bump version strings.
- Add a changelog file.
- Update strings that were previously pointing to thebeast.
- Update installation instructions

NOT READY FOR MERGE, let's wait until we resolve #1134.

### CODE REVIEW

File: qiita_db/environment_manager.py [hunk 1]
```diff
@@ -231,6 +231,11 @@ def make_environment(load_ontologies, download_reference, add_demo_user):
             ('demo@microbio.me', 4,
              '$2a$12$gnUi8Qg.0tvW243v889BhOBhWLIHyIJjjgaG6dxuRJkUM8nXG9Efe',
              'Demo', 'Qitta Dev', '1345 Colorado Avenue', '303-492-1984')""")
+        conn.execute("""
```
Comment (squirrelo) [inline qiita_db/environment_manager.py]: Oh, you already did this.

**Comment by squirrelo** [comment]
#1134 Resolved, should be good to go once tests pass

**Comment by squirrelo** [comment]
:+1: and tested on my machine

**Comment by josenavas** [comment]
NOOOOOOOO

**Comment by josenavas** [comment]
it was not working in a fully clean installation. @antgonza why did you close #1134 it was not solved! The problem is actually in setup.py. @ElDeveloper will need to create a new PR...

**Comment by antgonza** [comment]
Sorry ...

**Comment by ElDeveloper** [comment]
... I'll submit another PR ...

**Pull request merged**